app/views/sprint-7/monitor/cases/end-of-service-report.html

```diff
@@ -0,0 +1,79 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Manage intervention referrals
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+  {% include "../../includes/primary-navigation.html" %}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-third">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-0">
+          {{ serviceUser.name }}
+        </h1>
+        {% set currentPage = intervention.name %}
+        {% include "./side-navigation.html" %}
+        {% include "./intervention-sidebar.html" %}
+      </div>
+
+      <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-tabs" data-module="govuk-tabs">
+          <ul class="govuk-tabs__list">
+            <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+              <a class="govuk-tabs__tab" href="/sprint-7/monitor/cases/{{referral.reference}}/interventions/{{intervention.id}}#progress">
+                Progress
+              </a>
+            </li>
+            <li class="govuk-tabs__list-item">
+              <a class="govuk-tabs__tab" href="/sprint-7/monitor/cases/{{referral.reference}}/interventions/{{intervention.id}}#paper-trail">
+                Paper trail
+              </a>
+            </li>
+            <li class="govuk-tabs__list-item">
+              <a class="govuk-tabs__tab" href="/sprint-7/monitor/cases/{{referral.reference}}/interventions/{{intervention.id}}#statistics">
+                Statistics
+              </a>
+            </li>
+          </ul>
+
+          <div class="govuk-tabs__panel govuk-tabs__panel" id="intervention-progress">
+            <a href="/sprint-7/monitor/cases/{{referral.reference}}/interventions/{{intervention.id}}#progress" class="govuk-back-link govuk-!-margin-bottom-6">Back to Progress</a>
+            <h2 class="govuk-heading-l">
+              End of service report
+            </h2>
+            <p class="govuk-body">
+              The service provider has created an end of service report for {{ serviceUser.name }}’s {{ intervention.name | lower }} intervention. Please view the following end of service report.
+            </p>
+
+            <h2 class="govuk-heading-m">Outcomes</h2>
+
+            <h2 class="govuk-heading-s">Outcome 1 - Achieved</h2>
+            <p class="govuk-body">
+              {{ serviceUser.name }} has achieved her outcome 1 perfectly.
```

Comment:
Only thing I'd say is remove "her" from this line (and outcome 2 below) so it can work for any gender

Reply:
Thanks for spotting it! That's almost the first line I added and I totally forgot to revisit it after I realised how this page works :D